### ISSUE TITLE
Add Landing Page Description

### ISSUE THREAD

**Issue by haojianzong** [metrue/laivei#1]
Describe what this project does.

**Comment by metrue** [comment]
Yes, We should have a clear understanding of what we want to, and focus on one point that we really gonna do, which make our service to be different with the other ones.

**Comment by haojianzong** [comment]
So, the single, most important question we should ask ourselves is, what problem does this product solve?

**Comment by haojianzong** [comment]
如我们之前讨论的，产品有两个方面的用户群，
探索者：付出一定金钱，获得最独特的当地旅游体验。（探索者是否愿意/有多愿意付出这部分金钱，是关键）
表达者：通过对所在地方的独特经验，获取金钱收益。

**Comment by metrue** [comment]
是的。

我们现阶段的目标是解释好我们的服务，然后去看看有多少人愿意付出价格获取服务。
所以咱们目前就重点设计一下我们的landing page就好了。

landing page 需要：

1. 我们是否决定使用 探索者 和 表达者 这两个词语。
   如果是，如何让audience一下子明白他们的意义呢

2. 如何在写好我们的文案

3. 如何在 landing page 中展现我们的文案。

在 2015年5月13日 下午1:49，haojianzong <notifications@github.com>写道：

> 如我们之前讨论的，产品有两个方面的用户群，
> 探索者：付出一定金钱，获得最独特的当地旅游体验。（探索者是否愿意/有多愿意付出这部分金钱，是关键）
> 表达者：通过对所在地方的独特经验，获取金钱收益。
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/metrue/laivei/issues/1#issuecomment-101521372>.
>



-- 
Sincerely,

********************************************
www.minghe.me <Living@www.minghe.me>
********************************************
